nemo_skills/pipeline/utils.py

```diff
@@ -762,7 +776,8 @@ def add_task(
     executors = []
     # assuming server always has the largest resources request, so it needs to go first
     if server_config is not None:
-        server_cmd, num_server_tasks = get_server_command(**server_config, cluster_config=cluster_config)
+        server_port = server_config.pop("server_port", 5000)
```

Comment:
```suggestion
        server_port = server_config["server_port"]
```
I think we should have an error if port is not present as it's going to be pretty much required in many cases, so best if it's always explicitly provided